gateway-ha/src/main/java/io/trino/gateway/proxyserver/ProxyResponseHandler.java

```diff
@@ -38,7 +39,7 @@ public ProxyResponse handleException(Request request, Exception exception)
     public ProxyResponse handle(Request request, Response response)
     {
         try {
-            return new ProxyResponse(response.getStatusCode(), response.getHeaders(), toByteArray(response.getInputStream()));
+            return new ProxyResponse(response.getStatusCode(), response.getHeaders(), new String(toByteArray(response.getInputStream()), StandardCharsets.UTF_8));
```

Comment:
https://github.com/google/guava/blob/master/guava/src/com/google/common/io/ByteStreams.java#L234-L243
> <p><b>Java 9+ users:</b> use {@code in#readAllBytes()} instead.

I think we should use `readAllBytes()` instead of `toByteArray()`.